bench/dashboard/styles.css

```
@@ -0,0 +1,251 @@
+  --bg: #050505;
+  --surface: #0a0a0a;
+  --surface-2: #111111;
+  --border: #212121;
+  --border-light: #2d2d2d;
+  --text: #ededed;
+  --text-dim: #a2a2a2;
+  --text-muted: #6a6a6a;
+  --accent: #d97757;
+  --mono: "JetBrains Mono", "SF Mono", monospace;
+  --sans: "Inter", -apple-system, sans-serif;
+  --serif: "DM Serif Display", Georgia, serif;
```

Stylelint (17.3.0)

[error] Expected "Georgia" to be "georgia" (value-keyword-case)

(value-keyword-case)